packages/cf-wallet.js/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
+    "sourceRoot": ".",
```

Comment:
Why is this added?

Reply:
For parity w/ cf.js. I'd removed it from cf-wallet.js when I wrote it initially, since it was leading to a build error. This pr includes a fix for that build error, and so I could bring this line back. That being said, cf-wallet.js seems to work fine without it. Same with cf.js. Perhaps we could just remove it from both.

Reply:
Why do we need it?

Reply:
As far as I can tell, we don't use it at present because we're not actually using the sourcemaps we generate, at least not within the context of the playground. That being said, even if we were using sourcemaps, it looks like it's just the tsconfig default value. I'm not sure why we added it in the first place, but it's probably safe to remove.

Reply:
Went ahead and removed this line just to unblock things.